enterprise_learner_portal/api/v1/serializers.py

```diff
@@ -70,8 +70,8 @@ def to_representation(self, instance):
         representation['due_dates'] = get_due_dates(request, course_run_id, user)
         representation['pacing'] = course_overview['pacing']
         representation['org_name'] = course_overview['display_org_with_default']
-
-        representation['marked_done'] = instance.marked_done
+        representation['marked_done'] = instance.saved_for_later
```

Comment:
Do we need to keep this old field around? If it's still being used in the API might be worth adding a comment about it being deprecated

Reply:
It's still used by the frontend. The plan is to have a brief downtime where we switch the frontend over, then remove this in step 3.